cumulus/parachains/integration-tests/emulated/common/src/constants.rs

```diff
@@ -70,6 +70,10 @@ pub mod accounts {
 	pub const FERDIE_STASH: &str = "Ferdie//stash";
 	pub const FERDIE_BEEFY: &str = "Ferdie//stash";
 
+	pub fn get_treasury_address() -> AccountId {
+		PalletId(*b"py/trsry").into_account_truncating()
```

Comment:
is there no possible way to _NOT_ hardcode this?

Reply:
Its hardcoded in the runtime too, we can make it a constant though?

Reply:
Yes please.

Reply:
I think we dont need to add this here,
it is only used for `init_balances` where we drip some common accounts.

I think if any test needs treasury account with some balance, it should be as a part of preparing data for that concrete test. And then that concrete test can use runtime-specific Treasury account.

So I guess we dont need to specify `PalletId(*b"py/trsry")` here and also we dont need to even touch integration-tests in this PR. So I would revert changes from this file.

Reply:
removed in latest commit